docs/JSON_format.md

```diff
@@ -387,6 +387,7 @@ array item has the following attributes:
 - `id`: a unique integer (unique with respect to other parse value-sets)
 - `compressed_bitwidth`: the bitwidth of the values which can be added to the
 set. Note that this bitwidth does not include any padding.
+- `size` - number of elements in the set.
```

Comment:
comparing it with the rest of the JSON attributes (e.g. for tables) I now wish we had called it `max_size` for consistency; I believe we should still make that change

Reply:
The p4c IR definition for P4ValueSet uses `size`.  So does the P4-16 spec.  However, I see that JSON format uses max_size.  I'd be happy to change p4c to use max_size. 

Regarding "I would rather not merge a PR that only documents the new attribute without implementing support for it"

Are you saying the behavioral model does not support it yet?  If that is correct, I can put this PR on hold.

Reply:
See this p4c PR for maz_size JSON output.

https://github.com/p4lang/p4c/pull/2195

Reply:
If it supported it already, I would have updated this document as well. There is an open issue for this https://github.com/p4lang/behavioral-model/issues/856